addons/web_editor/static/src/js/backend.js

```diff
@@ -405,7 +405,19 @@ var FieldTextHtml = widget.extend({
                 var layoutInfo = this.editor.rte.editable().data('layoutInfo');
                 $.summernote.pluginEvents.codeview(undefined, undefined, layoutInfo, false);
             }
-            this.editor.buildingBlock.clean_for_save();
+            if (this.$iframe.is(':visible') || !$.browser.mozilla) {
```

Comment:
I would not do the mozilla check, just add a comment which says it is especially useful for mozilla.